codegen-server/python/src/main/kotlin/software/amazon/smithy/rust/codegen/server/python/smithy/generators/PythonApplicationGenerator.kt

```diff
@@ -179,4 +188,69 @@ class PythonApplicationGenerator(
             }
         }
     }
+
+    private fun renderPyApplicationRustDocs(writer: RustWriter) {
+        writer.rust(
+"""
+##[allow(clippy::tabs_in_doc_comments)]
+/// Main Python application, used to register operations and context and start multiple
+/// workers on the same shared socket.
+///
+/// Operations can be registrered using the application object as a decorator (`@app.operation_name`).
+///
+/// Here's a full example to get you started using coroutines:
```

Comment:
Seems like `operationImplementationStubs` will print non-async signatures? So it might confuse people with "started using coroutines" here?

Reply:
Very good point!